.github/workflows/regression.yml

```diff
@@ -753,9 +755,11 @@ jobs:
       - analyze-experiment
     env:
       FAILED: ${{ contains(needs.*.result, 'failure') }}
+      SKIP_REGRESSION: "${{ contains(github.event.pull_request.labels.*.name, 'ci-condition: skip-regression') }}"
+
     steps:
       - name: (PR review) Submit PR result as failed
-        if: github.event_name == 'pull_request_review' && env.FAILED == 'true'
+        if: github.event_name == 'pull_request_review' && env.FAILED == 'true' && env.SKIP_REGRESSION == 'true'
```

Comment:
I'm a bit confused by this and the below change. We want to set the commit status if it failed _and_ the PR was labeled to have regression tests skipped?